### ISSUE TITLE
Re: Repacking FileBot

### ISSUE THREAD

**Issue by rednoah** [Phoenix09/filebot#3]
The only change to the FileBot code is removing the donate nag on quit.

@Phoenix09 since you didn't bother to ask about repacking my code, nor intend to contribute anything meaningful back to the FileBot project, I'm kinda not cool with this.

You're welcome to repackage FileBot, especially for unsupported platforms such as Windows 7 / Windows Server, but I would kindly ask you *not* to remove the Donation Reminder in releases you make public. I mean, it's only really enabled for Windows 10 users anyway, so be nice please.

**Comment by td75** [comment]
rednoah thanks for your code but the donate nag was very annoying especially after already making donations. I really appreciate that Phoenix09 has taken the time to remove this and make it available to Windows 7 users again.

**Comment by rednoah** [comment]
The logic for nag screens is as follows:
https://www.filebot.net/forums/viewtopic.php?f=9&t=4866#p27192

Windows 7 users won't see nag screens unless they rename more than 2.000 files. That limit is raised exponentially every time the users presses *Donate* (regardless of whether or not he/she actually does so).

If you use FileBot so heavily that you actually see the nag screen, then a donation (the amount if which is up to you, may I remind you) is probably in order and a matter of just being polite.

You may always build FileBot yourself and make any modification you want, but if you want to publish release builds (which are obviously used by lots of Windows 10 users that don't want to pay for software) to the majority of non-tech-savvy potential customers, then we have a bit conundrum here.

**Comment by NHellFire** [comment]
Generally, a donate nag makes it less likely users will donate (even if they wanted to) because it's annoying. A fixed bar at the top would be preferred.
And users probably don't like $20 on the Windows store.. they'll either get it free or use something else.

**Comment by rednoah** [comment]
> Generally, a donate nag makes it less likely users will donate (even if they wanted to) because it's annoying.

Donations increased by more than 10x when I added the Donation Reminders some years ago. Your made-up claim does not match my raw data.

> they'll either get it free or use something else.

They can use different tools. That's perfectly fine. Just not an unofficial "free" version of FileBot.

**Comment by rednoah** [comment]
@Phoenix09 TL;DR Please remove the release packages from this repository. You're welcome to maintain your own fork for your own builds for you and your friends, just not the whole world.

**Comment by NHellFire** [comment]
The GPL disagrees with that. It prohibits additional restrictions. Everyone
is free to modify and distribute as they choose.

On 4 Jan 2018 15:25, "Reinhard Pointner" <notifications@github.com> wrote:

@Phoenix09 <https://github.com/phoenix09> TL;DR Please remove the release
packages from this repository. You're welcome to maintain your own fork for
your own builds for you and your friends, just not the whole world.

—
You are receiving this because you commented.
Reply to this email directly, view it on GitHub
<https://github.com/Phoenix09/filebot/issues/3#issuecomment-355310360>, or mute
the thread

**Comment by rednoah** [comment]
I'm kindly asking @Phoenix09 to stop publishing releases for FileBot as I am the original author of all the code and a fellow developer.

// asking as a friend
